scanpy/plotting/_anndata.py

```diff
@@ -576,15 +577,29 @@ def ranking(
                            top=1-(n_rows-1)*bottom-0.1/n_rows)
     for iscore, score in enumerate(scores.T):
         pl.subplot(gs[iscore])
-        indices = np.argsort(score)[::-1][:n_points+1]
+        if not include_lowest:
+            indices = np.argsort(score)[::-1][:n_points+1]
+        else:
+            sorted_scores = np.argsort(score)[::-1]
+            indices = sorted_scores[:(n_points//2)]
+            neg_indices = sorted_scores[-(n_points-(n_points//2)):]
         for ig, g in enumerate(indices):
             pl.text(ig, score[g], labels[g], color=color,
                     rotation='vertical', verticalalignment='bottom',
                     horizontalalignment='center', fontsize=8)
+        if include_lowest:
+            pl.text(len(indices), (score[g]+score[neg_indices[0]])/2, '⋮', color=color,
+                    rotation='vertical', verticalalignment='bottom',
+                    horizontalalignment='center', fontsize=8)
+            for ig, g in enumerate(neg_indices):
+                pl.text(ig+len(indices)+2, score[g], labels[g], color=color,
+                        rotation='vertical', verticalalignment='bottom',
+                        horizontalalignment='center', fontsize=8)
```

Comment:
Please use [black line wrapping](https://black.readthedocs.io/en/stable/the_black_code_style.html#how-black-wraps-lines) here and above